notifications.bs

```diff
@@ -1,26 +1,12 @@
-<pre class="metadata">
-Title: Notifications API Standard
+<pre class=metadata>
 Group: WHATWG
 H1: Notifications API
 Shortname: notifications
-Status: LS
-No Editor: true
+Text Macro: TWITTER notifyapi
 Abstract: This standard defines an API to display notifications to the end user, typically outside the top-level browsing context's viewport. It is designed to be compatible with existing notification systems, while remaining platform-independent.
-Logo: https://resources.whatwg.org/logo-notifications.svg
-Boilerplate: omit feedback-header, omit conformance
-!Participate: <a href=https://github.com/whatwg/notifications>GitHub whatwg/notifications</a> (<a href=https://github.com/whatwg/notifications/issues/new>new issue</a>, <a href=https://github.com/whatwg/notifications/issues>open issues</a>)
-!Participate: <a href=https://wiki.whatwg.org/wiki/IRC>IRC: #whatwg on Freenode</a>
-!Commits: <a href="https://github.com/whatwg/notifications/commits">GitHub whatwg/notifications/commits</a>
-!Commits: [SNAPSHOT-LINK]
-!Commits: <a href="https://twitter.com/notifyapi">@notifyapi</a>
-!Tests: <a href=https://github.com/w3c/web-platform-tests/tree/master/notifications>web-platform-tests notifications/</a> (<a href=https://github.com/w3c/web-platform-tests/labels/notifications>ongoing work</a>)
-!Translation (non-normative): <span title="Simplified Chinese"><a href=https://w3c-html-ig-zh.github.io/notifications/whatwg/ lang=zh hreflang=zh rel=alternate>简体中文</a></span>
+Translation: zh-cn https://w3c-html-ig-zh.github.io/notifications/whatwg/
```

Comment:
Is a more specific code better? (The resource itself seems to declare "en"...)

Reply:
In this case yes, as zh-cn is specifically Simplfied Chinese (which the resource is). It might be good to file an issue on the resource to update its language code, indeed.

Reply:
https://github.com/w3c-html-ig-zh/notifications has no way to file issues (and also seems like it's not actively maintained).

Reply:
Should we remove it?

Reply:
I'll leave it for now, on the logic that separate translation changes should stay separate.